app/templates/me/index.hbs

```diff
@@ -30,9 +30,9 @@
     </div>
 
     <p>
-        If you want to use package commands from the command line, you'll need a
-        <code>~/.cargo/credentials</code>, the first step to creating this
-        is to generate a new token using the button above.
+        If you want to use package commands from the command line, you will need to
+        login with <code>cargo login</code> using one of the tokens listed below. <br/>
+        When working in shared environments, supplying the token on the command line could expose it to other users, so it is more secure to supply a token when prompted. 
```

Comment:
Hmmm.... I'm thinking we could rephrase this to state the difference in using the `cargo login` command on workstations and in shared environments, since I feel entering the token on the 1 line is more convenient if safe to do so :)

So on line 34, you could state `<code> cargo login "token" </code>`, and on line 35, you could say "When working in shared environments, supplying the token via the aforementioned method could expose it to other users via the command line history. To avoid this, enter `cargo login` without the token, and enter the token when prompted to do so". Also, could we please keep the lines at a consistent length? Makes it easier to read in code editors and IDEs :).

How does this sound?